packages/preact-router/src/useNavigate.tsx

```
@@ -0,0 +1,62 @@
+import {
+  useContext,
```

ESLint

[error] 'useContext' is defined but never used.

(unused-imports/no-unused-imports)